src/main/java/io/github/sskorol/utils/ReflectUtils.java

```diff
@@ -0,0 +1,18 @@
+package io.github.sskorol.utils;
+
+import static org.joor.Reflect.on;
+
+/**
+ * Object creating processing class.
+ * Use it when you will create object via reflection.
+ */
+public final class ReflectUtils {
+
+    private ReflectUtils() {
+        throw new UnsupportedOperationException("Illegal access to private constructor.");
+    }
+
+    public static <T> T getObjectByArgs(final Class<T> entityClass, final Object... args) {
```

Comment:
try to rephrase it, as technically you're creating new instance not only by args... give it a more generic name...